distribution/docs/src/main/resources/_contents/introduction-contents.adoc

```diff
@@ -28,8 +28,8 @@ ${ddf-branding} Security Application::
 _${ddf-branding} version ${ddf.version}_ Provides authentication, authorization, and auditing services for the ${branding}.
 It is both a framework that developers and integrators can extend and a reference implementation that meets security requirements.
 
-${ddf-branding} Solr Catalog Application::
-_${ddf-branding} version ${ddf.version}_ Include the Solr Catalog Provider (SCP), an implementation of the `CatalogProvider` interface using http://lucene.apache.org/solr/[Apache Solr] as a data store.
+${ddf-branding} Solr Application::
```

Comment:
No that's some sort of adoc thing. Not sure what it does, but it doesn't show as a colon in the docs.